pywinauto/base_wrapper.py

```diff
@@ -861,31 +861,20 @@ def wait_for_idle(self):
         pass # do nothing by deafault
         # TODO: implement wait_for_idle for backend="uia"
 
-    #-----------------------------------------------------------
-    def type_keys(
-        self,
-        keys,
-        pause = None,
-        with_spaces = False,
-        with_tabs = False,
-        with_newlines = False,
-        turn_off_numlock = True,
-        set_foreground = True):
-        """
-        Type keys to the element using keyboard.SendKeys
-
-        This uses the re-written keyboard_ python module where you can
-        find documentation on what to use for the **keys**.
-
-        .. _keyboard: pywinauto.keyboard.html
-        """
+    # -----------------------------------------------------------
+    def _type_keys(
+            self,
+            keys,
+            func_to_call,
+            **kwargs):
         self.verify_actionable()
         friendly_class_name = self.friendly_class_name()
+        try:
+            kwargs['pause'] = kwargs['pause'] or Timings.after_sendkeys_key_wait
+        except KeyError:
+            pass
```

Comment:
Empty exception handler is not a good practice. In this case `kwargs.get('pause', Timings.after_sendkeys_key_wait)` looks better. Also I agree to allow `pause=0.0` explicitly (to one's own risk).

Reply:
Thanks for your review. Will make changes as you sugested.